additional-docs/example-configurations.md

```diff
@@ -44,6 +44,30 @@ environment variables. It's convenient, since you can keep the `s3_website.yml`
 in a public Git repo, and thus have your deployment configurations
 version-controlled.
 
+## Setup for HTTP2 and Custom SNI SSL Certificate
+
+To fully utilize HTTP2 you'll need to setup SSL for your distribution. While HTTP/2 does
+not mandate the use of encryption, it turns out that all of the common web browsers 
+require the use of HTTPS connections in conjunction with HTTP/2.
```

Comment:
Can you add a reference here? This is news to me, and I assume that it is news to some other users as well.